Source/Forms/MainForm.vb

```diff
@@ -1627,7 +1627,7 @@ Public Class MainForm
         Dim batchProject = ObjectHelp.GetCopy(Of Project)(p)
         batchProject.BatchMode = True
         batchProject.SourceFiles = {sourcefile}.ToList
-        Dim jobPath = batchFolder + sourcefile.Dir.Replace("\", "-").Replace(":", "-") + " " + p.TemplateName + " - " + sourcefile.FileName + ".srip"
+        Dim jobPath = Path.Combine(batchFolder, sourcefile.Dir.Replace("\", "-").Replace(":", "-") + " " + p.TemplateName + " - " + sourcefile.FileName + ".srip")
```

Comment:
shouldn't "\" be replaced with https://learn.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-7.0

Reply:
That's right, I've fixed it.  👍